meta/saisanitycheck.c

```diff
@@ -1279,7 +1279,20 @@ void check_attr_conditions(
 
         if (cmd->conditiontype != SAI_ATTR_CONDITION_TYPE_NONE)
         {
-            META_MD_ASSERT_FAIL(md, "conditional attribute is also conditional, not allowed");
+            if (md->objecttype == SAI_OBJECT_TYPE_MIRROR_SESSION &&
```

Comment:
please remove those exceptions from here, since conditional attributes cant be conditional event with exceptions

Reply:
Mirror session UDP src/dst port has to be set only when the tunnel is SFLOW and  Mirror session gre type has to be set only when the tunnel is L3_GRE.

Can you clarify on how this can be handled ?

Reply:
Use @validonly tag to specify UDP is only valid for SFLOW type of mirror

Reply:
@kcudnik can you check now please ?